cmd/server/assets/realm.html

```diff
@@ -323,7 +334,7 @@ <h1>Realm settings</h1>
           <div class="form-group row">
             <label for="twilio_auth_token" class="col-sm-3">Twilio auth token:</label>
             <div class="input-group col-sm-9">
-              <input type="password" id="twilio_auth_token" name="twilio_auth_token" class="form-control" value="{{if $smsConfig}}{{$smsConfig.TwilioAuthToken}}{{end}}">
+              <input type="password" id="twilio_auth_token" name="twilio_auth_token" class="form-control" autocomplete="new-password" value="{{if $smsConfig}}{{$smsConfig.TwilioAuthToken}}{{end}}">
```

Comment:
What's this do?

Reply:
prevents chrome from autofilling my username/password into this field

Reply:
Shouldn't it be `autocomplete="off"`

Reply:
Chrome aggressively ignores autocomplete="off". Don't know why.

https://stackoverflow.com/questions/57367813/2019-chrome-76-approach-to-autocomplete-off

It's been the cause of much internet lament. Looks like folks started using non-standard values like autocomplete="nope" to try to trick it, but now it ignores that too.

https://bugs.chromium.org/p/chromium/issues/detail?id=468153#c164

Reply:
That's the saddest thing.